libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -35,8 +35,6 @@
 
   public static final String EPSG_4326_URN = "urn:ogc:def:crs:EPSG::4326";
 
-  public static final String EPSG_X_4326_URN = "urn:x-ogc:def:crs:EPSG:4326";
```

Comment:
Are we removing this because it is not being used?  I wonder if it is being used in any downstream projects.

Reply:
I had originally added it for DDF-3552, and now I don't need it anymore. I doubt any downstream project has had time to start using it.